Samples/iOS-Swift/iOS-Swift/AppDelegate.swift

```diff
@@ -154,6 +154,33 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 }
                 return scope
             }
+            
+            options.configureUserFeedback = {
```

Comment:
Could we split the options that are optional from what's required, some comment blocks would be enough.
Just would love to see what is required first, to get an idea of minimal required setting (we could also do this on a different sample, and have this one show all settings)

Reply:
I think you're suggesting we have a couple different examples of a configuration, one completely default, one minimal, one highly customized?

I'm not sure I follow required vs. optional, as I currently think about all the config settings as optional. You can set the integration, define no options, and get a completely stock experience.

Reply:
Looking at how I'm going to do the docs, I think a collection of examples is better served there. The goal of this snippet in the sample app, for the time being anyways, is to make sure that all the sub-areas of configurations settings function correctly (the main config class, and the auxiliary classes, since i have them split up into general/widget/form/theme settings).